engine/common/rpc/convert/executor_metadata_test.go

```diff
@@ -0,0 +1,79 @@
+package convert_test
+
+import (
+	"testing"
+
+	"github.com/onflow/flow/protobuf/go/flow/entities"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/engine/common/rpc/convert"
+	"github.com/onflow/flow-go/model/access"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+func TestMessageToExecutorMetadata_Populated(t *testing.T) {
```

Comment:
you can actually simply these tests by converting back and forth. See other converters for examples
https://github.com/onflow/flow-go/blob/illia-malachyn/7652-fork-aware-events-endpoint/engine/common/rpc/convert/execution_results_test.go#L14-L26